src/utils/registration-helpers.ts

```diff
@@ -693,9 +695,34 @@ export const willResetNviStatuses = (persistedRegistration: Registration, update
   }
 
   const hasChangedCategory =
-    persistedRegistration.entityDescription?.reference?.publicationInstance.type !==
-    updatedRegistration.entityDescription?.reference?.publicationInstance.type;
+    persistedRegistration.entityDescription?.reference?.publicationContext.type !==
+    updatedRegistration.entityDescription?.reference?.publicationContext.type;
   if (hasChangedCategory) {
     return true;
   }
+
+  const persistedRegistrationWithJournal = persistedRegistration as JournalRegistration | ChapterRegistration;
+  const updatedRegistrationWithJournal = updatedRegistration as JournalRegistration | ChapterRegistration;
+  const hasChangedJournalId =
+    persistedRegistrationWithJournal.entityDescription?.reference?.publicationContext?.id !==
+    updatedRegistrationWithJournal.entityDescription?.reference?.publicationContext?.id;
+  if (hasChangedJournalId) {
+    return true;
+  }
+
+  const persistedRegistrationWithBook = persistedRegistration as BookRegistration;
+  const updatedRegistrationWithBook = updatedRegistration as BookRegistration;
```

Comment:
disse _har_ ikke bøker, men de _kan_ _være_ bøker. lett å gå seg vill 😅 Uansett endring på publihser vi er interessert i
```suggestion
  const persistedRegistrationWithPublisher = persistedRegistration as BookRegistration;
  const updatedRegistrationWithPublisher = updatedRegistration as BookRegistration;
```